pkgs/applications/graphics/openimageio/2.x.nix

```
@@ -16,13 +16,13 @@
 
 stdenv.mkDerivation rec {
   pname = "openimageio";
-  version = "2.2.17.0";
+  version = "2.4.6.0";
```

There is already a more recent tag, but it is not mentioned on the release page: https://github.com/OpenImageIO/oiio/tags

```suggestion
  version = "2.4.6.1";
```

(Don't forget to change the commit message and PR title accordingly.)

---

Yeah I was a little unclear as to whether they consider that actually "released".